building-apk.md

```diff
@@ -4,17 +4,17 @@ title: Building an APK for Android
 permalink: /building-apk/
 ---
 
-_Note: An `apk/AndroidManifest.xml` file was already created for
-your app by `flutter create`. You can check it out if you like,
-but you don't need to modify it when getting started._
+Follow these steps to build a deployment-ready APK:
 
-First, ensure you installed the Android SDK tools
-(see "Setting up your Android device" from [Getting started](/getting-started)).
+1. cd into your application's directory
+1. Run `flutter build apk`
 
-Run `android update sdk` from the command line,
-and install SDK version 22 and build tools version 22.0.1.
-For now, Flutter currently expects only _these_ versions.
+The APK can be found in `MY_FLUTTER_APP/build/app.apk`.
 
-Then, inside your Flutter app, run `flutter build apk`.
+# Installing an APK onto a device
 
-The APK can be found in `$MY_FLUTTER_APP/build/app.apk`.
+Follow these steps to install your APK onto a connected Android device:
+
+1. Connect your Android device to your workstation/laptop
+1. cd into your application's directory
+1. Run `flutter install`
```

Comment:
Yeah, except that on the second run, you're not left in a state where you can just unplug your phone and go. On the second run of `flutter run`, the updated flx is put into a tmp directory (it doesn't actually update the APK).